### PR TITLE
Make the library do nothing

### PR DESCRIPTION
**Pull request by rowanmanning**
This is a breaking change because the library no longer sends any metrics to Graphite. The entire public API has been replaced with properties and methods that do nothing except log a warning.

This means:

  * Systems can stop sending metrics without having to immediately remove all imports of next-metrics and calls to `count` etc

  * Once we're ready we can mark this major version as "safe" and just watch all the dependency PRs get merged instead of teams having to manually change anything

  * If this library is still installed it's _way_ smaller and has fewer dependencies to worry about

Some things to consider:

  * We may want to release patches to the 12.x branch if security issues crop up. This is fine and relatively easy to do - we'll just need to manually tag releases in npm to stop 12.x versions being `latest`

  * We'll also want a corresponding major version of n-express that uses this version of next-metrics.

### CODE REVIEW

File: test/unit/lib/metrics.spec.js
Comment (apaleslimghost) [inline test/unit/lib/metrics.spec.js:1]: **suggestion**: is it worth replacing this with an integration test to make sure v13 definitely has the same API surface as v12?

File: index.js [hunk 1]
```diff
@@ -1,10 +1,85 @@
...
+ * @param {function} classFn
+ * @param {string} propertyName
+ * @param {any} value
+ */
+function addDeprecatedProperty (classFn, propertyName, value) {
+	Object.defineProperty(classFn.prototype, propertyName, {
+		get: buildDeprecationLogger(`${classFn.name}#${propertyName}[get]`, value),
+		set: buildDeprecationLogger(`${classFn.name}#${propertyName}[set]`, value)
+	});
+}
```
Comment (apaleslimghost) [inline index.js:45]: **praise**: neat!

**Review by apaleslimghost — APPROVED** [review]
lfg

File: test/unit/lib/metrics.spec.js
Comment (rowanmanning) [reply test/unit/lib/metrics.spec.js:1]: Probably does but not sure if I can be bothered 😂 is that bad?

Comment (apaleslimghost) [reply test/unit/lib/metrics.spec.js:1]: yolo

**Review by CyntiBinti — APPROVED** [review]

**Pull request merged**